libevmcore/Assembly.h

```diff
@@ -24,6 +24,7 @@
 #include <iostream>
 #include <sstream>
 #include <libdevcore/Common.h>
+#include <libdevcore/SourceLocation.h>
```

Comment:
I see your point but I don't wholy agree. We should have a languages related repository/directory where all these generic language related functionalities would go. Or rather as you suggest rename evmcore. It is misleading.

In anycase for now I will move it to evmcore